### PR TITLE
Add automap Recipe

### PR DESCRIPTION
**Pull request by brandtbucher**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by brandtbucher** [comment]
@conda-forge-admin, the AppVeyor failure seems unrelated:

```
error: command 'cl.exe' failed: No such file or directory
```

Same error is happening over on #11261.

**Comment by beckermr** [comment]
You can ignore the appveyor error

**Comment by brandtbucher** [comment]
@beckermr so is this good to go, then?

File: recipes/automap/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  home: https://github.com/brandtbucher/{{ name }}
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.md
+  summary: 'High-performance autoincremented integer-valued mappings.'
+  dev_url: https://github.com/{{ author }}/{{ name }}
+
+extra:
+  recipe-maintainers:
+    - {{ author }}
```
Comment (beckermr) [inline recipes/automap/meta.yaml:40]: This is going to break the bot and other parts. Please remove the author jinja2 variable

**Comment by brandtbucher** [comment]
Thanks!

**Pull request merged**